.github/workflows/create-design-review-issue.js

```diff
@@ -145,11 +151,81 @@ async function postGithub(token, url, data) {
   return JSON.parse(body);
 }
 
+async function graphqlQueryGithub(token, query) {
+  const {data} = await postGithub(token, '/graphql', {query});
+  return data;
+}
+
+async function postGithub(token, url, data) {
+  return requestGithub(token, url, data, {method: 'POST'});
+}
+
 function postGithubIssue(token, repo, data) {
-  const url = `https://api.github.com/repos/${repo}/issues`;
+  const url = `/repos/${repo}/issues`;
   return postGithub(token, url, data);
 }
 
+function closeGithubIssue(token, repo, number) {
```

Comment:
Two suggestions to aid maintainability and code quality:

1. Move this file from `.github/workflows/` to `build-system/common/`. This will automatically enable lint checks and prettier auto-formatting.
2. Add minimal JSDoc for all functions in this file to provide type info. This will result in automatic type-checking of this code via `amp check-build-system`.

If you'd like, this can be done via a separate PR.

Reply:
Done.